.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
We dont want launch configs for the api in the repo, they were purposefully left out. we should remove the entire root .vscode